### ISSUE TITLE
Test suite missing files in sdist tarball

### ISSUE THREAD

**Issue by snoyberg** [dhess/hpio#47]
Likely just needs some extra-source-files or something like that.

```
> /tmp/stackage-build8/hpio-0.8.0.0$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup configure --enable-tests --package-db=clear --package-db=global --package-db=/home/stackage/work/builds/nightly/pkgdb --libdir=/home/stackage/work/builds/nightly/lib --bindir=/home/stackage/work/builds/nightly/bin --datadir=/home/stackage/work/builds/nightly/share --libexecdir=/home/stackage/work/builds/nightly/libexec --sysconfdir=/home/stackage/work/builds/nightly/etc --docdir=/home/stackage/work/builds/nightly/doc/hpio-0.8.0.0 --htmldir=/home/stackage/work/builds/nightly/doc/hpio-0.8.0.0 --haddockdir=/home/stackage/work/builds/nightly/doc/hpio-0.8.0.0 --flags=
Configuring hpio-0.8.0.0...
> /tmp/stackage-build8/hpio-0.8.0.0$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup build
Building hpio-0.8.0.0...
Preprocessing library hpio-0.8.0.0...
[ 1 of 12] Compiling System.GPIO.Types ( src/System/GPIO/Types.hs, dist/build/System/GPIO/Types.o ) [dist/build/autogen/cabal_macros.h changed]
Preprocessing executable 'hpio-sysfs-example' for hpio-0.8.0.0...
Linking dist/build/hpio-sysfs-example/hpio-sysfs-example ...
Preprocessing executable 'hpio-example' for hpio-0.8.0.0...
Linking dist/build/hpio-example/hpio-example ...
Preprocessing executable 'hpio-reader-example' for hpio-0.8.0.0...
Linking dist/build/hpio-reader-example/hpio-reader-example ...
Preprocessing test suite 'hlint' for hpio-0.8.0.0...
[1 of 1] Compiling Main             ( test/hlint.hs, dist/build/hlint/hlint-tmp/Main.o )
Linking dist/build/hlint/hlint ...
Preprocessing test suite 'doctest' for hpio-0.8.0.0...
[1 of 1] Compiling Main             ( test/doctest.hs, dist/build/doctest/doctest-tmp/Main.o )
Linking dist/build/doctest/doctest ...
Preprocessing test suite 'spec' for hpio-0.8.0.0...
[1 of 1] Compiling Main             ( test/Main.hs, dist/build/spec/spec-tmp/Main.o )

test/Main.hs:4:1: error:
    Failed to load interface for ‘Spec’
    Use -v to see a list of the files searched for.
```

**Comment by dhess** [comment]
Ahh, right. `hspec` can discover `.hs` test files in lieu of putting them in `build-depends`, but I still need to tell cabal about them. Thanks for the report.

**Comment by dhess** [comment]
@snoyberg This error didn't show up in my Travis-CI Stackage tests. My `.travis.yaml` is based on the one in the Stackage docs, but am I missing something that should have caught this?

**Comment by snoyberg** [comment]
Older versions of that Travis config wouldn't have caught such an issue, but newer versions should.

**Comment by dhess** [comment]
Fixed by b6dc5a0ea3bc3a2d58f749ddd94e824c50d8594c.
